### DISCUSSION TITLE
teachers screen larger than clients

### DISCUSSION THREAD

**Discussion by nektarios1201** [epoptes/epoptes#80] [Q&A]
The teachers screen is larger than client. So when broadcast it clients see only part of the screen and need to move the mouse to see the rest of it.
Is there a way to make clients see the whole screen by scale down teachers screen without changing the resolution on teachers pc?

thanks
Nektarios

**Comment by alkisg** [comment]
If you install ssvnc, its viewer will be used, and it supports scaling.
But it's best to have the same resolution, students see the broadcast more clearly.
Or you can run epoptes in a virtual machine or under xephyr, so that it's inside a smaller window that matches the client resolutions.

**Comment by nektarios1201** [comment]
Hi Alkis
Sorry for the delay.
The problem for the ssvnc scaling is that the user should press F8 and choose the scaling.
I wanted to do it from teacher's pc.
So finally I decide to change the resolution on teacher's pc to match students.
Thanks a lot
